File: 2003/IntroToVBNETPart2/Presentation/StaticVariables.vb
Imports System

Class Test
    Shared Sub RememberLastItem(ByVal item As String)
        Static lastItem As String = ""
        Console.WriteLine("item: '{0}' last item: '{1}'", item, lastItem)
        lastItem = item
    End Sub
    Shared Sub Main()
        RememberLastItem("Apple")
        RememberLastItem("Orange")
        RememberLastItem("Banana")
    End Sub
End Class

File: 2003/IntroToVBNETPart2/Presentation/StaticVariables.cs
using System ;
class Test
{
     static  string lastItem  = "" ;

     static void RememberLastItem ( string item )
     {
          Console.WriteLine("item: '{0}' last item: '{1}'", item, lastItem) ;
          lastItem = item ;
     }
     public static void Main()
     {
          RememberLastItem("Apple") ;
          RememberLastItem("Orange") ;
          RememberLastItem("Banana") ;
     }
}
